bin/alias.bat

```diff
@@ -12,7 +12,7 @@ if not ["%_temp%"] == ["%_temp2%"] (
 	endlocal
 	goto:eof
 )
-
+echo.>> "%CMDER_ROOT%\config\aliases"
```

Comment:
Without the newline, new alias will be added after the exist one, and then everything is error. :(